### DISCUSSION TITLE
sha256 checksum of install-pdm.py does not match

### DISCUSSION THREAD

**Discussion by bmcandr** [pdm-project/pdm#1958] [General]
The sha256 checksum for `install-pdm.py` currently shown on [the website](https://pdm.fming.dev/latest/) does not match the file. The website says the checksum of the current `install-pdm.py` file should be `ce0a116987b2667231391d13dd005006433114033cac74aa18f0b2dec5538d03`.

![Screen Shot 2023-05-28 at 11 43 52 AM](https://github.com/pdm-project/pdm/assets/19274445/58b7a2e9-c207-4a48-9292-326d482b84a4)

Generating the checksum locally or checking with an online tool returns a different sha256 checksum:

```
$ curl -sSL https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py | shasum -a 256
06abd94a6678636eba640529bf91b242759363c45d3620cdfabaa0053d826b30  -
```

![Screen Shot 2023-05-28 at 11 50 46 AM](https://github.com/pdm-project/pdm/assets/19274445/f34036b5-e5da-407e-93c2-1956bf366b8e)

([source](https://emn178.github.io/online-tools/sha256_checksum.html))

**Comment by frostming** [comment]
See #1942
